docs/development/releasing.rst

```diff
@@ -466,22 +546,28 @@ Backporting fixes from main
 
 .. note::
 
-    The changelog script in ``astropy-tools`` (``pr_consistency`` scripts
-    in particular) does not know about minor releases, thus please be careful.
+    The changelog script in `astropy-tools <https://github.com/astropy/astropy-tools/>`_
```

Comment:
Should we remove all instructions about these scripts from here and just have the instructions in the astropy-tools repo and link to those?

Reply:
Ok done, I've opened https://github.com/astropy/astropy-tools/pull/168 to add a little to the astropy-tools instructions